docs/robots.txt

```diff
@@ -1,5 +1,12 @@
 User-agent: *
 Allow: /en/latest/ # Allow all robots to access the latest version of the documentation
-Disallow: /en/ # Disallow all robots to access the other versions of the documentation
+Allow: /en/release-4.0/ # Allow robots to access version 4.0, which is a newer version
```

Comment:
Excellent!

Reply:
Should we disallow release-4.0 as well since we want google to only parse latest version?

Reply:
> Should we disallow release-4.0 as well since we want google to only parse latest version?

yes agreed, will make changes

Reply:
Agreed!!